scripts/zones/Leujaoam_Sanctum/npcs/Mining_Point.lua

```diff
@@ -0,0 +1,84 @@
+-----------------------------------
+-- Area: Leujaoam Sanctum
+-- NPC: Nulwahah
+-- Assault: Orichalcum Survey
+-----------------------------------
+local ID = require("scripts/zones/Leujaoam_Sanctum/IDs")
+require('scripts/globals/settings')
+require("scripts/globals/assault")
+-----------------------------------
+local entity = {}
+
+entity.onTrigger = function(player, npc)
+    -- Offset to match worms to mining points
+    local strikes = npc:getLocalVar("strikeCounter")
+    local instance = npc:getInstance()
+    local offset = 152
+
+    if npc:checkDistance(player) > 3 then
+        player:messageSpecial(ID.text.MOVE_CLOSER)
+        return
+    end
+
+    if not xi.assault.hasTempItem(player, xi.items.PICKAXE) then
+        player:messageSpecial(ID.text.MINING_POSSIBLE, xi.items.PICKAXE)
+        return
+    end
+
+    if GetMobByID(npc:getID() - offset, instance):isSpawned() then
+        player:messageSpecial(ID.text.CANT_MINE)
+        return
+    end
+
+    if strikes < 8 then
+        player:sendEmote(GetNPCByID(npc:getID(), instance), xi.emote.EXCAVATION, xi.emoteMode.MOTION)
```

Comment:
I think you have the wrong order of parameters here. Check the sendEmote function again. On my phone but it looks like the wrong order

Reply:
kk will do

Reply:
Order and the call is being done 100% the same as in helm.lua